restapi/src/main/java/io/stargate/web/models/Keyspace.java

```diff
@@ -28,6 +29,7 @@
   @JsonProperty("datacenters")
   List<Datacenter> datacenters;
 
+  @ApiModelProperty(value = "")
```

Comment:
Should this be documented describing how it affects replication factor in each of the DCs?